main.go

```diff
@@ -12,32 +12,38 @@ import (
 
 	"github.com/alphagov/plek/go"
 	"github.com/alphagov/publishing-api/controllers"
+	"github.com/alphagov/publishing-api/errornotifier"
 	"github.com/alphagov/publishing-api/request_logger"
 )
 
 var (
-	arbiterHost          = plek.Find("url-arbiter")
-	liveContentStoreHost = getEnvDefault("CONTENT_STORE", "http://content-store.dev.gov.uk")
-	port                 = getEnvDefault("PORT", "3093")
-	requestLogDest       = getEnvDefault("REQUEST_LOG", "STDOUT")
+	arbiterURL          = plek.Find("url-arbiter")
+	liveContentStoreURL = getEnvDefault("CONTENT_STORE", "http://content-store.dev.gov.uk")
+	port                = getEnvDefault("PORT", "3093")
+	requestLogDest      = getEnvDefault("REQUEST_LOG", "STDOUT")
 
-	draftContentStoreHost = os.Getenv("DRAFT_CONTENT_STORE")
+	draftContentStoreURL = os.Getenv("DRAFT_CONTENT_STORE")
 
 	renderer = render.New(render.Options{})
+
+	errbitHost     = os.Getenv("ERRBIT_HOST")
+	errbitApiKey   = os.Getenv("ERRBIT_API_KEY")
+	errbitEnvName  = os.Getenv("ERRBIT_ENVIRONMENT_NAME")
+	errbitNotifier = errornotifier.NewErrbitNotifier(errbitHost, errbitApiKey, errbitEnvName)
```

Comment:
How will this behave if the ENV vars aren't defined (eg in dev).  Might it be better to move the construction of this into `main`?